packages/animations/test/fade_scale_transition_test.dart

```diff
@@ -71,8 +71,10 @@ void main() {
       // Opacity duration: First 30% of 150ms, linear transition
       double topFadeTransitionOpacity = _getOpacity(key, tester);
       double topScale = _getScale(key, tester);
+      Alignment topScaleAlignment = _getScaleAlignment(key, tester);
```

Comment:
Since we don't anticipate the `topScaleAlignment` to change throughout the animation (it doesn't really make sense for it to), maybe it's better to revert these changes to the existing tests and add one, simpler test that verifies that the `ScaleTransition`'s alignment was set to the appropriate value when the modal is shown.